packages/shared/src/dto/playback.ts

```
@@ -0,0 +1,36 @@
+
+import type { ITelemetryData } from "../types";
```

ESLint

[error] Delete `⏎`

(prettier/prettier)